Source/MQTTnet.AspnetCore/SpanBasedMqttPacketBodyReader.cs

```diff
@@ -0,0 +1,125 @@
+﻿using MQTTnet.Exceptions;
```

Comment:
Is it possible to use this kind of Formatter also in the base lib and just use the old version for .NET 4.5.2 etc. (where not supported)?

Reply:
If you are ok with a new dependency to System.Memory and System.Buffers i can move this to the core lib

Reply:
After reading the article you mentioned I want to test the ArrayPool to get rid of this _singleByteBuffer_ and _fixedHeaderBuffer_ etc. If this all gives us at least a moderate performance or memory improvement I am fine with it.